doc/source/user_guide/ug_case_settings.rst

```diff
@@ -369,6 +369,9 @@ forcing_nml
    "", "``minus1p8``", "constant ocean freezing temperature (:math:`-1.8^{\circ} C`)", ""
    "", "``mushy``", "matches mushy-layer thermo (ktherm=2)", ""
    "``trestore``", "integer", "sst restoring time scale (days)", "90"
+   "``cpl_frazil``", "``external``", "frazil water/salt fluxes are handled outside of Icepack", "``fresh_ice_correction``"
+   "", "``internal``", "send full frazil water/salt fluxes for mushy physics", ""
+   "", "``fresh_ice_correction``", "correct fresh-ice frazil water/salt fluxes for mushy physics", ""
```

Comment:
These should be alphabetical to be consistent with the rest of the document (hopefully).  So external, fresh_ice_correction, then internal.  Should just have to swap the order of internal and fresh_ice_correction is you want to follow that format.